content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -5,20 +5,19 @@ weight: 1
 keywords: [kubernetes]
 ---
 
-To install and configure Istio in a Kubernetes
-cluster, follow these instructions:
+To install and configure Istio in a Kubernetes cluster, follow these instructions:
 
 ## Prerequisites
 
 1. [Download the Istio release](/docs/setup/kubernetes/download-release/).
 
-1. Kubernetes platform setup
-  * [Minikube](/docs/setup/kubernetes/minikube/)
-  * [Google Container Engine (GKE)](/docs/setup/kubernetes/gke/)
-  * [IBM Cloud Kubernetes Service (IKS)](/docs/setup/kubernetes/ibm/)
-  * [Openshift Origin](/docs/setup/kubernetes/openshift/)
-  * [Amazon Web Services (AWS) with Kops](/docs/setup/kubernetes/aws/)
-  * [Azure](/docs/setup/kubernetes/azure/)
+1. [Kubernetes platform setup](/docs/setup/kubernetes/platform-setup/):
```

Comment:
This link does not work in netlify, let's remove it, it's not really needed since we have all the other links.